pkg/apis/meta/v1alpha1/net.go

```diff
@@ -9,49 +10,95 @@ import (
 
 // PortMapping defines a port mapping between the VM and the host
 type PortMapping struct {
-	HostPort uint64 `json:"hostPort"`
-	VMPort   uint64 `json:"vmPort"`
+	BindAddress net.IP   `json:"bindAddress,omitempty"`
```

Comment:
this is optional; I'd prefer for this to be a pointer (although, technically maybe omitempty works here)

Reply:
`net.IP` is a `[]byte`, which can be `nil`, so omitempty works just fine.